packages/ubuntu_wsl_setup/lib/args_common.dart

```diff
@@ -14,9 +14,8 @@ screens, yet allowing user to overwrite any of those during setup.
 }
 
 List<String>? serverArgsFromOptions(ArgResults options) {
-  if (options['prefill'] == null) {
-    return null;
-  }
-
-  return ['--prefill', options['prefill']];
+  return <String>[
+    if (options['prefill'] != null) ...['--prefill', options['prefill']],
+    if (options['reconfigure'] != null) '--autoinstall=',
```

Comment:
this might deserve a comment to explain what and why :)

Reply:
Done :D